### PR TITLE
Using CreditAnswer type

### PR DESCRIPTION
**Pull request by lmcewanhmrc**

### CODE REVIEW

**Review by liampan — APPROVED** [review]

File: app/navigation/ReturnsJourneyNavigator.scala [hunk 1]
```diff
@@ -107,9 +107,10 @@ class ReturnsJourneyNavigator @Inject()(
     controllers.returns.credits.routes.ConfirmPackagingCreditController.onPageLoad(mode)
 
   def confirmCreditRoute(mode: Mode): Call =
-    if (mode.equals(CheckMode))
-      routes.ReturnsCheckYourAnswersController.onPageLoad()
-    else
+    // TODO disable for now - otherwise can try to jump to full CYA on first pass through journey -> crash
```
Comment (liampan) [inline app/navigation/ReturnsJourneyNavigator.scala:110]: think we have an idea of how to solve this, I can impliment today?

File: app/pages/returns/credits/ConvertedCreditsWeightPage.scala [hunk 2]
```diff
@@ -19,6 +19,7 @@ package pages.returns.credits
 import pages.QuestionPage
 import play.api.libs.json.JsPath
 
+// TODO axe
```
Comment (liampan) [inline app/pages/returns/credits/ConvertedCreditsWeightPage.scala:22]:
```suggestion
@deprectated("use blah", "new credits 1 year")
```

**Review by liampan — APPROVED** [review]

File: app/navigation/ReturnsJourneyNavigator.scala (hunk 1 above)
Comment (liampan) [reply app/navigation/ReturnsJourneyNavigator.scala:110]:
```
def trueCreditsCheckMode(userAnswers: UserAnswers, mode: Mode) = {
    mode == CheckMode &&
    userAnswers.get(ManufacturedPlasticPackagingWeightPage).isDefined &&
    userAnswers.get(ImportedPlasticPackagingWeightPage).isDefined &&
    userAnswers.get(DirectlyExportedWeightPage).isDefined &&
    userAnswers.get(AnotherBusinessExportedWeightPage).isDefined &&
    userAnswers.get(NonExportedHumanMedicinesPlasticPackagingWeightPage).isDefined &&
    userAnswers.get(NonExportedRecycledPlasticPackagingWeightPage).isDefined
  }
```

Comment (lmcewanhmrc) [reply app/navigation/ReturnsJourneyNavigator.scala:110]: Cool, thanks. New branch tho?

File: app/pages/returns/credits/ConvertedCreditsWeightPage.scala (hunk 2 above)
Comment (lmcewanhmrc) [reply app/pages/returns/credits/ConvertedCreditsWeightPage.scala:22]: Working on tidying this on new branch

**Pull request merged**